src/component/handlers/edit/editOnBeforeInput.js

```diff
@@ -185,6 +185,17 @@ function editOnBeforeInput(
         parentNode.firstChild.nodeType === Node.TEXT_NODE &&
         parentNode.firstChild.nodeValue.indexOf('\t') !== -1;
     }
+    // If selected block was empty, DraftEditorTextNode rendered it as <BR>.
+    // But now we typed some chars to it, so <BR> should be removed.
+    // We must let React render this properly.
+    if (
+      nativeSelection.anchorNode &&
+      nativeSelection.anchorNode.nodeName === 'SPAN' &&
+      nativeSelection.anchorNode.firstChild.nodeName == 'BR' &&
+      chars
```

Comment:
`chars` check is unnecessary (it's already checked on line 97)

also… does it make sense to check that `.firstChild === .lastChild` too?